settings.gradle

```diff
@@ -131,6 +131,9 @@ if (!System.getenv().containsKey("SUB_BUILD") || System.getenv().get("SUB_BUILD"
     include ':airbyte-integrations:connectors:destination-gcs'
 
     include ':tools:code-generator'
+
+    include 'airbyte-integrations:connectors-performance:source-harness'
+    findProject(':airbyte-integrations:connectors-performance:source-harness')?.name = 'source-harness'
```

Comment:
what is this `findProject` for?

Reply:
This was created by IntelliJ's module manager.
Doesn't appear to be necessary. removed
thanks